R/build_projection_model.r

```diff
@@ -0,0 +1,64 @@
+#' Model future uptake from current uptake
+#'
+#' BEWARE: THIS DOES NOT YET ACCOUNT FOR MULTI-YEAR OR MULTI-STATE DATA SETS
+#'
+#' Future incident uptake can be predicted from current incident uptake.
+#' A model can be built and fit to project incident uptake into the future.
+#'
+#' The default model says that current incident uptake is a linear function
+#' of the previous incident uptake, time elapsed since rollout, and their
+#' interaction.
+#'
```

Comment:
Elapsed is the time (in days) since the previous report. Incident is the change in uptake (in population %) since the previous report. Daily is the change in uptake (in population %) _per day_ since the previous report.

So daily = incident / elapsed.